Tasks/ArchiveFiles/archivefiles.ts

```diff
@@ -268,15 +264,15 @@ function doWork() {
                 try {
                     var stats: tl.FsStats = tl.stats(archiveFile);
                     if (stats.isFile()) {
-                        console.log('removing existing archive file before creation: ' + archiveFile);
+                        console.log(tl.loc('RemoveBefore', archiveFile));
                     } else {
-                        failTask('Specified archive file: ' + archiveFile + ' already exists and is not a file.');
+                        failTask(tl.loc('ArchiveFile', archiveFile));
```

Comment:
Again, this key is also too generic. Consider something like "ArchiveFileExistsButNotAFile"?